src/prop/minisat/core/Solver.h

```diff
@@ -252,8 +252,14 @@ class Solver {
     // Mode of operation:
     //
     int       verbosity;
+    // for LRB
+    double step_size;
+    double step_size_dec;
+    double min_step_size;
```

Comment:
It looks like these three variables are not actually initialized. Am I missing something?

Reply:
Note: if my comment is correct and the variables are not initialized, we probably need to reevaluate the strategies affected by it.

Reply:
Yeah, I think you are correct that they are not initialized. This is probably why LRB did not win much. I think I need to re-evaluate two strategies: lrb and lrb+stoponly. The other ones should be fine.